libraries/ui-library/src/components/six-textarea/six-textarea.tsx

```diff
@@ -104,9 +106,11 @@ export class SixTextarea {
   @Prop() autofocus = false;
 
   /** The textarea's spellcheck attribute. */
+  // eslint-disable-next-line @stencil-community/reserved-member-names
   @Prop() spellcheck = false;
 
   /** The textarea's inputmode attribute. */
+  // eslint-disable-next-line @stencil-community/reserved-member-names
```

Comment:
Definitely not a fan of adding eslint-disable comments in so many places. Maybe it makes more sense to completely remove this rule alltogether then?

Reply:
Me neither. I think it's fine to turn off this specific rule. But I suggest we keep `@stencil-community/ban-default-true` for which I disabled some in the code too, as it is helpful to prevent more boolean properties that default to true.